### PR TITLE
Gp 2023.02.11

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Ciel/Day 11/11_01_CIEL11_2_1.txt [hunk 1]
```diff
@@ -392,6 +392,7 @@ You got lucky and reincarnated into a brat with some kinda vampire-like ability,
 [sha:bad1e43bcd59a2ef393587560673c7b81fb71718]{
 -- Page 117, Offset 29400.
 -- 悲惨な話だがね。この街には、一心同体の人でなしが二人もいた事になる」
+// Come back to this one - change to you, past since one is dead, better way to encode 一心同体?
```
Comment (rschlaikjer) [inline script/Ciel/Day 11/11_01_CIEL11_2_1.txt:395]: Kinda rearranged, but 
> What a fuckin' mess. Even though you've only got one soul, there were still two damn copies of you running about. 

?

File: script/Ciel/Day 11/11_01_CIEL11_2_2.txt [hunk 2]
```diff
@@ -178,7 +178,7 @@ Who am I to criticize, when I'm here, feeding in such a place?
 [sha:4b53c3722b3ba72fa8bf03e29f261819332af8fd]{
 -- Page 129, Offset 29449.
 -- 自虐ついでに過去の記録を閲覧する。
-// Not even vampires can escape cringe
+// 自虐 referring to prev sentence
```
Comment (rschlaikjer) [inline script/Ciel/Day 11/11_01_CIEL11_2_2.txt:181]: > My self-derisive musings spur me to think back on my past.

?

File: script/Ciel/Day 12/12_01_CIEL12_2.txt [hunk 3]
```diff
@@ -63,6 +63,7 @@ Everything I had before still remains, just as it was.
 [sha:109cb61993a571aa6b531de343ad74c7d3285a02]{
 -- Page 23, Offset 30923.
 -- そしてそれが、先輩の残してくれた唯一の<成|も><果|の>だった。
+// Flag, may be hard to grasp initially
```
Comment (rschlaikjer) [inline script/Ciel/Day 12/12_01_CIEL12_2.txt:66]: Kinda think it's okay as is, but alt suggestion that makes it clearer
> In a sense, this return to my ordinary life is the one thing Senpai left behind.

File: script/Ciel/Day 12/12_01_CIEL12_2B.txt [hunk 4]
```diff
@@ -428,7 +430,7 @@ Shouldn't have said that...
 -- Page 58, Offset 31011.
 -- 「……すみません、あまりに先輩が神出鬼没なんで、良からぬ想像をする時もなくはありませんでした。先輩の耳には届いてないだろうけど、一、二年の間じゃ楽しい噂話もあるし」
 "I'm sorry, but given how liable you are to pop up in the most unlikely places,
- you can't fault people for coming up with the odd wild and crazy theory about you.
+ you can't fault people for coming up with the odd wild and crazy t needs another lookheory about you.
```
Comment (rschlaikjer) [inline script/Ciel/Day 12/12_01_CIEL12_2B.txt:433]: Oops editing error, revert

File: script/Ciel/Day 12/12_01_CIEL12_3_2B.txt [hunk 5]
```diff
@@ -108,6 +108,7 @@ Kohaku-san briskly heads off toward the west wing, where her room is located.
 [sha:e29b66015700620face678866ffe44713c478406]{
 -- Page 201, Offset 31418. Mods: @n.
 -- 失礼します、と断ってから、
+// Make it clear that this isn't referring to the prev line
```
Comment (rschlaikjer) [inline script/Ciel/Day 12/12_01_CIEL12_3_2B.txt:111]: > Hisui proceeds to apologize in advance, then timidly lends me her shoulder.

File: script/Ciel/Day 12/12_01_CIEL12_2.txt (hunk 3 above)
Comment (Avuxo) [reply script/Ciel/Day 12/12_01_CIEL12_2.txt:66]: I like the alt suggestion but I also don't think the cur. version is all that hard to grasp. There's no page break after all

File: script/Ciel/Day 12/12_01_CIEL12_2B.txt [hunk 6]
```diff
@@ -367,7 +367,9 @@ Senpai managed to beat me to the stairs somehow.
 [sha:032e1f80685fcb8c06d8734dfdaf90a9ce37def5]{
 -- Page 53, Offset 31000.
 -- いつも待ってるばかりだから、たまにはこっちから行こうと思ったんです」
-You're always the one doing the visiting, so I figured I'd stop by your classroom for a change."
+// Weird one needs another look
+I feel like I always have to wait too long to see you again,
```
Comment (Avuxo) [inline script/Ciel/Day 12/12_01_CIEL12_2B.txt:371]: slightly altered meaning
> I feel like I'm always the one waiting for you to come to me, so I figured I'd take the initiative once in a while.

**Review by Avuxo — APPROVED** [review]
nice work today fellas, sorry i couldnt attend

File: script/Ciel/Day 11/11_01_CIEL11_1C.txt [hunk 7]
```diff
@@ -393,7 +393,7 @@ I kick the chair. Who wouldn't?
 [sha:7da3cccbe3c4127ac680205f884c82e4f61c5b69]{
 -- Page 74, Offset 29214.
 -- すると、蹴った反動で足がよけいに痛んだ。
-Kicking the chair causes my leg to be assailed by even more unnecessary pain.
+Kicking the chair causes my leg to be assailed by even more superfluous pain.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 11/11_01_CIEL11_1C.txt:396]: Reason I changed this is to more clearly match an earlier superfluous in the scene - IIRC from the original tlc review, the chairs are superfluous -> pain is also superfluous / those chairs shouldn't be here -> I shouldn't have to feel this pain.

File: script/Ciel/Day 12/12_01_CIEL12_2B.txt (hunk 6 above)
Comment (rschlaikjer) [reply script/Ciel/Day 12/12_01_CIEL12_2B.txt:371]: Yeah that feels like more what he's actually saying to me, but I forget the reason we didn't go with something like that. Maybe Petrikow remembers.

File: script/Ciel/Day 11/11_01_CIEL11_1C.txt (hunk 7 above)
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 11/11_01_CIEL11_1C.txt:396]: gotcha, makes sense

**Review by yk1512 — APPROVED** [review]
🙏

File: script/Ciel/Day 12/12_01_CIEL12_3_1.txt [hunk 8]
```diff
@@ -156,6 +156,7 @@ Why do they act this way?
 [sha:3f7012fc4e60100f99c55bc9c915c018e97e253c]{
 -- Page 163, Offset 31292.
 -- 無為な昆虫のように、知性あるものが、命題もなく群れているのか。
+// Petrikow flag for contrast
 Why do these intelligent beings mill about, without purpose, like shiftless insects?
```
Comment (Petrikow) [inline script/Ciel/Day 12/12_01_CIEL12_3_1.txt:160]: Something like this was my thought...
Essentially, "why act like insect (dumb) when smart?"
>Why are they milling about, without purpose, like shiftless insects, when endowed with the gift of wisdom?

**Pull request merged**